BH/Modules/StashExport/StashExport.cpp

```diff
@@ -92,14 +91,31 @@ JSONObject* StashExport::getStatEntry(WORD statId, WORD statId2, DWORD statVal,
 	switch (statId) {
 	case STAT_SINGLESKILL:
 	case STAT_NONCLASSSKILL: {
-		auto sk = Tables::Skills.binarySearch("Id", statId2);
-		if (sk) {
-			entry->set("name", NAMEOF(statId));
-			entry->set("skill", sk->getString("skill"));
+		SkillsTxt* pSkillsTxt = &(*p_D2COMMON_sgptDataTable)->pSkillsTxt[statId2];
```

Comment:
The only issue with modifying the stash export is that if we ever want to use the ctrl + C copy item for pasting to trade functionality, we'll have to rewrite the parser on the site. Maybe it needs to be done either way though